src/plugins/StatusBar/StatusBar.js

```diff
@@ -136,16 +170,16 @@ const ProgressBarUploading = (props) => {
           ? html`<div title="Uploading">${pauseResumeButtons(props)} Uploading... ${throttledProgressDetails(props)}</div>`
           : html`<div title="Paused">${pauseResumeButtons(props)} Paused・${props.totalProgress}%</div>`
         : null
-        }
+      }
     </div>
   `
 }
 
 const ProgressBarComplete = ({ totalProgress, i18n }) => {
   return html`
-    <div class="UppyStatusBar-content">
+    <div class="UppyStatusBar-content" role="alert">
```

Comment:
`alert` is a role for very important information, i think the progress is not _that_ important—would `status` be more appropriate?

Reply:
But would you say alert is fine for errors though?